src/script/lua_api/l_noise.cpp

```diff
@@ -104,24 +104,25 @@ void LuaPerlinNoise::Register(lua_State *L)
 	registerClass(L, className, methods, metamethods);
 
 	lua_register(L, className, create_object);
+	lua_register(L, "PerlinNoise", create_object); // deprecated name
```

Comment:
It is probably better to be done in env.lua as `core.get_perlin `and include a deprecated message also.